zrml/prediction-markets/src/weights.rs

```diff
@@ -766,7 +766,7 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     /// Storage: MarketCommons Markets (r:0 w:1)
     /// Proof: MarketCommons Markets (max_values: None, max_size: Some(542), added: 3017, mode: MaxEncodedLen)
     /// The range of component `m` is `[0, 63]`.
-    fn create_market_and_deploy_pool(m: u32) -> Weight {
+    fn create_market_and_deploy_pool(m: u32, _n: u32) -> Weight {
```

Comment:
Because I didn't run the benchmarks correctly. If you configure this correctly, I think it'll be fine.

Reply:
Can you try to get this correct?

Reply:
I thought the idea was that we don't run the benchmarks in PRs. Did I get that right?

Reply:
I think it makes sense to run the benchmarks with low steps and reps beforehand to figure out that this case for example is prevented (unused benchmark component). Otherwise you wouldn't notice, or would you?

Reply:
66e5fc1b027317d010250b71050107f7b966046a